node/pkg/vaa/structs.go

```diff
@@ -58,6 +59,16 @@ type (
 	}
 
 	SignatureData [65]byte
+
+	TransferPayloadHdr struct {
+		Type         uint8
+		TokenChainID ChainID
+		TokenAddress Address
+		ToChainID    ChainID
+		ToAddress    Address
+		Amount       *big.Int
+		Fee          *big.Int
```

Comment:
Probably don't need / won't want the fee field after the payload 3 update, and I don't think we planned on using it anyway.

Reply:
I removed the fee.